Zend/zend_vm_def.h

```diff
@@ -10062,6 +10062,19 @@ ZEND_VM_HOT_TYPE_SPEC_HANDLER(ZEND_IS_NOT_EQUAL|ZEND_IS_NOT_IDENTICAL, (op1_info
 	ZEND_VM_SMART_BRANCH(result, 0);
 }
```

Comment:
Can this PR be tested using opcache instructions dump?

Reply:
The dump doesn't change. This only changes the handler.